_episodes/8.md

```diff
@@ -0,0 +1,128 @@
+discuss the journey in more detail of what led us to our current trajectory with
+the Presto® SQL project and why that is now being renamed to Trino. We also
+discuss how this affects those that are using Trino. If you are developing on
+Trino and have the old namespace check out the 
+[guide to migrate here](https://trino.io/blog/2021/01/04/migrating-from-prestosql-to-trino.html).
+
+We also discuss the differences between the two projects. It is actually a lot
+after two years since the split and we recommend looking at the end-of-year
+[blog we wrote at the end of 2019](https://trino.io/blog/2020/01/01/2019-summary.html)
+and keep your eyes peeled for the blog we are writing to summarize the changes
+in 2020!
+
+Finally, we cover some sneak peeks at the Roadmap for Trino in 2021.
+
+If you want to show us some 💕, please [give us a ⭐ on Github](https://github.com/trinodb/trino/blob/master/.github/star.png).
+
+
+## Events, news, and various links
+Blogs
+ - <https://trino.io/blog/2019/05/21/optimizing-the-casts-away.html>
+ - <https://towardsdatascience.com/statistics-in-spark-sql-explained-22ec389bf71b>
+ - <https://www.infoworld.com/article/3597971/on-premises-data-warehouses-are-dead.html>
+ - <https://www.javahelps.com/2019/11/presto-sql-join-algorithms.html>
+ 
+Upcoming events
+ - Feb 9 - Feb 10 <http://starburstdata.com/datanova>
+
+Latest training from David, Dain, and Martin(Now with timestamps!):
+ - <https://trino.io/blog/2020/07/15/training-advanced-sql.html>
+ - <https://trino.io/blog/2020/07/30/training-query-tuning.html>
+ - <https://trino.io/blog/2020/08/13/training-security.html>
+ - <https://trino.io/blog/2020/08/27/training-performance.html>
+
+Presto® Summit Series - Real world usage
+ - <https://trino.io/blog/2020/05/15/state-of-presto.html>
+ - <https://trino.io/blog/2020/06/16/presto-summit-zuora.html>
+ - <https://trino.io/blog/2020/07/06/presto-summit-arm-td.html>
+ - <https://trino.io/blog/2020/07/22/presto-summit-pinterest.html>
+
+Podcasts:
+ - [Presto with Martin Traverso, Dain Sundstrom and David Phillips](https://www.contributor.fyi/presto)
+ - [Simplify Your Data Architecture With The Presto Distributed SQL Engine](https://www.dataengineeringpodcast.com/presto-distributed-sql-episode-149/)
+ - [How Open Source Presto Unlocks a Single Point of Access to Data](https://redhatx.buzzsprout.com/755519/5980279)
+ - [The Data Access Struggle is Real](https://redhatx.buzzsprout.com/755519/5656471)
+ - [Presto with Justin Borgman](https://softwareengineeringdaily.com/2020/02/07/presto-with-justin-borgman/)
+ - [The infrastructure renaissance and how it will power the modernization of analytics platforms](https://redhatx.buzzsprout.com/755519/3923864)
+ - [Uber’s Data Platform with Zhenxiao Luo](https://softwareengineeringdaily.com/2018/05/24/ubers-data-platform-with-zhenxiao-luo/)
+ 
+
+If you want to learn more about Trino yourself, you should check out the 
+O’Reilly Presto Definitive guide. You can download 
```

Comment:
If you want to learn more about Trino, check out the definitive guide from OReilly.